.swiftlint.yml

```diff
@@ -1,14 +1,15 @@
 included:
-  - Survey
-  - SurveyTests
-  - SurveyUITests
+  - iosApp/Survey
+  - iosApp/SurveyTests
+  - iosApp/SurveyUITests
 
 excluded:
-  - SurveyTests/Sources/Mocks/Sourcery/AutoMockable.generated.swift
-  - Survey/Sources/Supports/Helpers/Rswift/R.generated.swift
-  - Pods
-  - Derived
-  - DerivedData
+  - iosApp/SurveyTests/Sources/Mocks/Sourcery/AutoMockable.generated.swift
+  - iosApp/Survey/Sources/Supports/Helpers/Rswift/R.generated.swift
+  - iosApp/Pods
+  - iosApp/Derived
+  - iosApp/DerivedData
+  - iosApp/vendor
```

Comment:
Just curious why do we need to move the the `.swiftlint.yml` file out of the `iosApp` folder and modified the included/excluded files like this? 🤔

Reply:
The whole reason is that I would like to use nested configuration for SwiftLint but:
1. Danger-swift-ruby [doesn't work](https://github.com/ashfurrow/danger-swiftlint/issues/4) well with nested configuration so we have to use `lint_all_files`.
2. CICD runs Danger on the root even with the tag `working-directory: iosApp`, with `lint_all_files` it can only read the `SwiftLint.yml` on the root of the project, hence we have to move the file to root and specify iosApp folder explicitly.

Reply:
@minhnimble You can see more fail attempts from this [PR history](https://github.com/blyscuit/kmm-swiftUI-IC/pull/95) if that intrigues you.

Reply:
I see, it looks like a limitation from Danger with nested configuration for SwiftLint and this is the workaround. Thanks for the insights 🙏